jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.py

```diff
@@ -208,6 +216,8 @@ def _set_display_unit_of_selected_dataset(self):
             return
 
         data = self.dataset.selected_dc_item
+        if isinstance(data, list):
+            data = data[0]

```

Comment:
don't love this, but this whole method will probably go away with #3156, and I can't think of a better way to do it with the current setup.

Reply:
cc @cshanahan1

Reply:
this will go away at some point (im trying to consolidate everything that listens to unit changes to one method in that PR), but it also seems like its necessary here for now